feconf.py

```diff
@@ -213,6 +213,15 @@
 # for recently published explorations in /library/recently_published page.
 RECENTLY_PUBLISHED_QUERY_LIMIT_FULL_PAGE = 20
 
+# The current version of the Android feedback report info blob schema.
+CURRENT_ANDROID_REPORT_SCHEMA_VERSION = 1
+
+# The current version of the web feedback report info blob schema.
+CURRENT_WEB_REPORT_SCHEMA_VERSION = 1
+
+# The current version of the app feedback report daily stats blob schema.
+CURRENT_REPORT_STATS_SCHEMA_VERSION = 1
```

Comment:
Per other PR, CURRENT_APP_FB_REPORT_STATS_SCHEMA_VERSION perhaps. Not so keen on the abbreviation but not sure how to otherwise make it shorter while avoiding the generic-ness of just "report".

Reply:
I'm worried FB will be too misconstrued with Facebook (which doesn't have much relevancy with Oppia but I think FB has become too common for Facebook to be useful here).

What about CURRENT_FEEDBACK_REPORT_STATS_SCHEMA_VERSION?

Reply:
I agree with your concern -- thanks for raising it. The name you propose sounds fine IMO.

Reply:
Done